addons/website_project_issue/models/project_issue.py

```diff
@@ -1,13 +1,29 @@
 # -*- coding: utf-8 -*-
 # Part of Odoo. See LICENSE file for full copyright and licensing details.
 
-from odoo import api, exceptions, models, _
+from odoo import api, fields, exceptions, models
 
 
 class Issue(models.Model):
     _name = "project.issue"
     _inherit = ['project.issue']
 
+    main_attachment_ids = fields.One2many('ir.attachment', compute='_compute_attachment_ids', string="Main Attachments",
+                                          help="Attachment that don't come from message.")
+    website_url = fields.Char('Website URL', compute='_compute_website_url', help='The full URL to access the document through the website.')
+
+    def _compute_website_url(self):
```

Comment:
There should be only one place with website URLs. I think you should extract it into a method and use it in compute website url and in get access action.